README.md

```diff
@@ -86,9 +86,48 @@ using insights-proxy. You may run the proxy using the SPANDX config provided her
 SPANDX_CONFIG=$(pwd)/compliance-backend.js ../insights-proxy/scripts/run.sh
 ```
 
-### Option 3: Docker Development setup
```

Comment:
I've also updated the readme with some more details about this environment.

Reply:
thanks @akofink